.github/workflows/create-jira-close-pr.yml

```
@@ -0,0 +1,56 @@
+  create-pr:
+    needs: run-test
+    name: Create PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download Codes
+        uses: actions/checkout@v4
+
+      # 브랜치 이름에서 Jira 이슈 번호 추출 (예: feat/#SCURM-1-login)
+      - name: Create PR Title
+        id: pr-title
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:19: Double quote to prevent globbing and word splitting

(shellcheck)